### ISSUE TITLE
Verovio does not respect page-breaks and system-breaks when converting MusicXML to SVG

### ISSUE THREAD

**Issue by apacha** [rism-ch/verovio#1832]
I'm trying to convert a MusicXML file to SVG files with Verovio, but unfortunately the page- and system-breaks are ignored and a warning is issued.

This is the input MusicXML file that I am using: [bach_bwv_1013_p10_12_mov_1.xml.zip](https://github.com/rism-ch/verovio/files/5547041/bach_bwv_1013_p10_12_mov_1.xml.zip)

I'm running Verovio using the following command: `verovio --breaks encoded --all-pages --to svg bach_bwv_1013_p10_12_mov_1.xml`

This produces the following output and two files without the original breaks:
```
[Warning] Requesting layout with encoded breaks but nothing provided in the data
Output written to bach_bwv_1013_p10_12_mov_1_001.svg.
Output written to bach_bwv_1013_p10_12_mov_1_002.svg.
```
That warning would make sense, if there were no page- and system breaks in the file, but they are there. 

Converting the same file to MEI with `verovio --breaks encoded --all-pages --to mei bach_bwv_1013_p10_12_mov_1.xml` produces a similar output:

```
[Warning] Requesting layout with encoded breaks but nothing provided in the data
Output written to bach_bwv_1013_p10_12_mov_1.mei.
```
However, the produced MEI file seems to include the breaks. When I render that MEI file to SVG now, it correctly produces three files: 

```
verovio --breaks encoded --all-pages --to svg bach_bwv_1013_p10_12_mov_1.mei
Output written to bach_bwv_1013_p10_12_mov_1_001.svg.
Output written to bach_bwv_1013_p10_12_mov_1_002.svg.
Output written to bach_bwv_1013_p10_12_mov_1_003.svg.
```

I'm currently using the latest development build from here: [Verovio 3.1.0-dev-3f1709c-dirty](https://github.com/eNote-GmbH/verovio/commit/3f1709cc5777528b4c2ffda6fe3bc2e80b5958a2) and also tested on official [3.0.1](https://github.com/rism-ch/verovio/commit/9be368d81d3298212ee31e0a9b827d9e4748100c).

**Comment by rettinghaus** [comment]
Verovio checks the metadata if the encoding actually supports page and system breaks. But your file lacks this information: 

```xml
 <identification>
    <encoding>
      <supports attribute="new-system" element="print" type="yes" value="yes"/>
      <supports attribute="new-page" element="print" type="yes" value="yes"/>
    </encoding>
  </identification>
```

Would be easy to change, but I think this approach is favourable. Opinions @craigsapp @lpugin ?

**Comment by craigsapp** [comment]
I would say that relying `identification/encoding` in such a way is not useful.  These are encoding hints and not encoding realities.  That information gives the state of the data rather than defines the state of the data.  In other words, if there is no such information, then the state of the data is undefined, and you have to look at all of the data to determine the state.  If `supports@value="no"`, then it would be reasonable to suppress the system/page breaks found in the data.  But if there is no `identification/encoding` section, then the default values for various features should be considered to be undefined, but can be assumed to be `yes`.

In other words if you see the element

```xml
<print new-page="yes"/>
```

Then it should be converted to `<pb>`.  This should probably be done regardless of any `supports` statement: if it is in the data, then it should be converted rather than stripped out.

**Comment by rettinghaus** [comment]
@craigsapp Actually it respects all `<print />` elements and converts them correctly, the `encoding` info is just used to set `m_hasLayoutInformation` as `true`.

**Comment by lpugin** [comment]
Anything that needs or can be done? Otherwise I will close the issue.

**Comment by rettinghaus** [comment]
As I said, would be easy to change, if we agree to completely and only rely on `--breaks` option for this.

**Comment by lpugin** [comment]
Would it be appropriate to set `m_hasLayoutInformation` to `true` if `<print>` tags are encountered and forget about `<encoding>`?
